mm2src/coins_activation/src/tendermint_with_assets_activation.rs

```diff
@@ -50,13 +66,10 @@ pub struct TendermintActivationParams {
     #[serde(default)]
     pub path_to_address: HDPathAccountToAddressId,
     #[serde(default)]
-    #[serde(deserialize_with = "deserialize_account_public_key")]
-    with_pubkey: Option<TendermintPublicKey>,
-    #[serde(default)]
-    is_keplr_from_ledger: bool,
+    pub activation_params: Option<TendermintPubkeyActivationParams>,
```

Comment:
I believe this is an API breaking change that will require changes in MoonFi GUI specifically. We can keep the breaking change but inform @KomodoPlatform/qa and MoonFi team @onur-ozkan or we can flatten this.